_posts/2020-01-05-working-offline-should-not-be-so-hard.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Working Offline Should Not Be So Hard
+date: 2020-01-05 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+I'm back trying to [work offline][1] and it is astonishing how many applications simply refuse to work when offline. Some kind of apps will never work offline, because they simply can't (skype, slack, ..) and that's totally warranted, that's the whole point of the application. What I'm more worried about are all the rest that do not need to have a server component at all, but apparently do and even more so refuse to work if they cannot connect to it.
```

Comment:
Over the holidays... lol

I STRIVE FOR ACCURACY

Reply:
Or back FROM

Reply:
No either about either...